src/lib/y2packager/product_finder.rb

```diff
@@ -184,6 +184,9 @@ def find_dependencies(product_solvable, selected_base)
       jobs = select_products(product_solvable, selected_base)
       solver = pool.Solver
 
+      # disable soft dependencies (Recommends), use only the hard dependencies (Requires)
+      solver.set_flag(Solv::Solver::SOLVER_FLAG_IGNORE_RECOMMENDED, 1)
```

Comment:
What is the lifetime of this flag setting?
Will it affect package resolving further on?
Is libzypp using the same solver instance as ruby-solv?

(maybe inconsequential as I want to cancel the whole PR)

Reply:
Libzypp uses a different solver instance, there is no side effect of this change.